### PR TITLE
style: line color and cell gap

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Mischa's review.

# Changes

- adjust cell gap spacing
- introduce line color

### CODE REVIEW

File: src/lib/styles/themes/dark.scss [hunk 1]
```diff
@@ -63,6 +63,10 @@
   --disable: #505050;
   --disable-contrast: #969696;
 
+  // Line
+  --line: #505050;
```
Comment (mstrasinskis) [inline src/lib/styles/themes/dark.scss:67]: Out of curiosity, where are we gonna use it?

**Review by mstrasinskis — APPROVED** [review]
LGTM.

File: src/lib/styles/themes/dark.scss (hunk 1 above)
Comment (peterpeterparker) [reply src/lib/styles/themes/dark.scss:67]: Mischa said "there are lots of lines in the design" and also that he use the term "line" in his Figma file.

Currently we use the line in the neuron of the proposal detail and we will use also the color as background of the detail (see screenshot in red)
Capture d’écran 2022-09-02 à 12 45 49

Comment (mstrasinskis) [reply src/lib/styles/themes/dark.scss:67]: Thank you for the update.

**Pull request merged**